go-apps/meep-tc-engine/go.sum

```diff
@@ -1,11 +1,9 @@
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
-github.com/KromDaniel/jonson v0.0.0-20180630143114-d2f9c3c389db h1:Zkf5kwhxdW0xV7WM/crqIcOP5LCFGnAmumWSFAewJ74=
+github.com/InterDigitalInc/AdvantEDGE v1.2.0 h1:46Mr4OqKz/6WU/1CziSacw62Z3pc52dgSOLppB5N/Bc=
```

Comment:
I assumed it appeared there because go.mod was changed to use the meep-redis package rather than calling it directly inside the tc-engine. This was the only difference in the go.mod file that could have created ripple effects in the go.sum